app/src/main/res/layout/beer_layout.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    >
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/cardView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:cardCornerRadius="@dimen/cardview_default_radius"
+        android:elevation="@dimen/cardview_default_elevation"
+        android:layout_marginTop="24sp"
+        android:layout_gravity="center"
+        style="@style/Base.CardView"
+        >
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ImageView
+                android:id="@+id/imagePath"
+                android:layout_width="88dp"
+                android:layout_height="139dp"
+                android:layout_marginStart="34sp"
+                android:layout_marginTop="22sp"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:srcCompat="@drawable/ichnusa"
+                />
+
+
+            <TextView
+                android:id="@+id/title"
+                android:layout_width="150dp"
```

Comment:
Questo dovrebbe essere 172dp, in base a quanto scritto su Figma. Controlla anche che gli altri valori siano corretti

Reply:
Ho sbagliato, però credo sia wrap_content più che altro.
La dimensione mi è rimasta li perchè il testo e mi andava oltre la card allora gli avevo dato un limite. :)